controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -815,6 +815,22 @@ private Throwable getRealException(Throwable ex) {
                 .thenApplyAsync(res -> TxnState.newBuilder().setState(TxnState.State.valueOf(res.name())).build(), executor);
     }
 
+    /**
+     * List transaction in completed(COMMITTED/ABORTED) state.
+     * @param scope scope name
+     * @param stream stream name
+     * @param requestId request id
+     * @return map having transactionId and transaction status
+     */
+    public CompletableFuture<Map<UUID, io.pravega.controller.store.stream.TxnStatus>> listCompletedTxns(final String scope, final String stream,
```

Comment:
can you import this class?

Reply:
We can't import this class. As it will conflict with already imported `io.pravega.controller.stream.api.grpc.v1.Controller.TxnStatus` class.